tools/generate_types_deno.ts

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env -S deno run -A
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+// This script is used to generate the @types/deno package on DefinitelyTyped.
+
+import $ from "jsr:@david/dax@0.42.0";
+import { Node, Project } from "jsr:@ts-morph/ts-morph@23.0.0";
+import * as semver from "jsr:@std/semver@1.0.3";
```

Comment:
Should we open an issue about using a single `deno.json` to store all dependencies in `tools/deno.json`?